src/Map/src/Bridge/Google/composer.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/ux-map-google",
+    "type": "symfony-ux-map-bridge",
+    "description": "Symfony UX Map Google Renderer",
```

```suggestion
    "description": "Symfony UX Map GoogleMaps Bridge",
```

wdyt ? (the name of their product is "Google Maps")

---

Yup